RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -2275,7 +2252,7 @@ boolean L11_unlockPyramid()
 				auto_log_warning("Did not have an Arid Desert Item and the Pyramid is next. Must backtrack and recover", "red");
 				if((my_adventures() >= 3) && (my_meat() >= 500))
 				{
-					doVacation();
+					LX_doVacation();
 					if(item_amount($item[Shore Inc. Ship Trip Scrip]) > 0)
 					{
 						cli_execute("make UV-Resistant Compass");
```

Comment:
Should replace this whole if statement with a call to your new `L11_getUVCompass()` function.

Reply:
refactored that entire function.